src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StoppedNode.kt

```diff
@@ -6,5 +6,6 @@ import com.atlassian.performance.tools.ssh.api.Ssh
 internal interface StoppedNode {
     val jiraHome: String
     val ssh: Ssh
+    val unpackedProduct: String
```

Comment:
All the configurations until now were done in `StoppedNode`. Is it possible to keep that convention? I have three ideas that could work here:
1. Can a Node accept a list of  `fun updateConfiguration(ssh, jiraHome, unpackedProduct)` functions and run it after the internal configuration?
2. Can we pass LB future to a stopped node? Does it make sense? 
3. Can we expose  `fun updateConfiguration(ssh, jiraHome, unpackedProduct)` instead of unpackedProduct?

@marcinsasinowski, @dagguh WDYT?

Reply:
I was thinking about options you presented and I will choose the 1st option (still might be premature, but let's do it :) )